### ISSUE TITLE
Support multiple tables in the 'analyze-table' argument

### ISSUE THREAD

**Issue by IanMeyers** [awslabs/amazon-redshift-utils#241]

**Comment by IanMeyers** [comment]
See https://github.com/awslabs/amazon-redshift-utils/issues/43 for original detail

**Comment by IanMeyers** [comment]
This is now supported from https://github.com/awslabs/amazon-redshift-utils/commit/7208a1e8729d67bfb8607448744d7f7c3cefc859

**Comment by wjordan** [comment]
@IanMeyers the [comment](https://github.com/awslabs/amazon-redshift-utils/commit/7208a1e8729d67bfb8607448744d7f7c3cefc859#commitcomment-26826868) in 7208a1e8729d67bfb8607448744d7f7c3cefc859 is correct, there are missing quotes around line 941 after this commit.

I reproduce the following error on latest master when analyzing a single table, using the environment variable `-e ANALYZE_TABLE=[table]` with the Docker entrypoint:

```
Running column-encoding utility
-- [8] Connected to [host]:[port]:[db] as [user]
-- [8] Analyzing Table '[table]' for Columnar Encoding Optimisations with 1 Threads...
Traceback (most recent call last):
  File "ColumnEncodingUtility/analyze-schema-compression.py", line 1193, in <module>
    main(sys.argv)
  File "ColumnEncodingUtility/analyze-schema-compression.py", line 1186, in main
    result_code = run()
  File "ColumnEncodingUtility/analyze-schema-compression.py", line 975, in run
    query_result = execute_query(statement)
  File "ColumnEncodingUtility/analyze-schema-compression.py", line 112, in execute_query
    cursor.execute(str)
  File "/usr/local/lib/python2.7/site-packages/pg8000/core.py", line 852, in execute
    self._c.execute(self, operation, args)
  File "/usr/local/lib/python2.7/site-packages/pg8000/core.py", line 1741, in execute
    self.handle_messages(cursor)
  File "/usr/local/lib/python2.7/site-packages/pg8000/core.py", line 1879, in handle_messages
    raise self.error
pg8000.core.ProgrammingError: (u'ERROR', u'42703', u'column "[table]" does not exist in a, pgc, unnamed_join, pgd, unnamed_join, pgn, unnamed_join, pgu, unnamed_join, b', u'/home/ec2-user/padb/src/pg/src/backend/parser/parse_expr.c', u'1540', u'transformColumnRef')
Done
```

Adding quotes as suggested in the comment fixes the error.

(Cross-referencing this issue to #229, since this error is related to a lack of automated testing).

**Comment by IanMeyers** [comment]
The error is being thrown because there’s no table by the name you supplied. The quoting in the log message looks correct. Can you please run with a valid table name and the debug option so we can see the sql generated?

Thx,

Ian

**Comment by wjordan** [comment]
> The error is being thrown because there’s no table by the name you supplied. The quoting in the log message looks correct.

No, the table definitely exists and the quoting is definitely incorrect. As I already mentioned, the query runs correctly after the quoting fix in #247 is applied.

Here is the invalid query generated by current master (with `[schema]` and `[table]` as placeholders):

```
# select trim(a.name) as table, b.mbytes, a.rows, decode(pgc.reldiststyle,0,'EVEN',1,'KEY',8,'ALL') dist_style, TRIM(pgu.usename) "owner", pgd.description
from (select db_id, id, name, sum(rows) as rows from stv_tbl_perm a group by db_id, id, name) as a
join pg_class as pgc on pgc.oid = a.id
left outer join pg_description pgd ON pgd.objoid = pgc.oid
join pg_namespace as pgn on pgn.oid = pgc.relnamespace
join pg_user pgu on pgu.usesysid = pgc.relowner
join (select tbl, count(*) as mbytes
from stv_blocklist group by tbl) b on a.id=b.tbl
and pgn.nspname = '[schema]' and pgc.relname in ([table]);
ERROR:  column "[table]" does not exist in a, pgc, unnamed_join, pgd, unnamed_join, pgn, unnamed_join, pgu, unnamed_join, b
```

Here is the valid, fixed query (note the extra quoting for `pgc.relname in ('[table]');`:
```
# select trim(a.name) as table, b.mbytes, a.rows, decode(pgc.reldiststyle,0,'EVEN',1,'KEY',8,'ALL') dist_style, TRIM(pgu.usename) "owner", pgd.description
from (select db_id, id, name, sum(rows) as rows from stv_tbl_perm a group by db_id, id, name) as a
join pg_class as pgc on pgc.oid = a.id
left outer join pg_description pgd ON pgd.objoid = pgc.oid
join pg_namespace as pgn on pgn.oid = pgc.relnamespace
join pg_user pgu on pgu.usesysid = pgc.relowner
join (select tbl, count(*) as mbytes
from stv_blocklist group by tbl) b on a.id=b.tbl
and pgn.nspname = '[schema]' and pgc.relname in ('[table]');
            table            | mbytes |   rows   | dist_style | owner | description 
-----------------------------+--------+----------+------------+-------+-------------
[...]
(1 row)
```

**Comment by IanMeyers** [comment]
Merged the PR cited.
